### PR TITLE
Update Chronograf docs for 0.11

### PR DESCRIPTION
**Pull request by rkuchan**
* Creates a new 0.11 section and makes it the default documentation for Chronograf
* Updates all images and gifs
* Makes minor edits to the Getting Started guide
* Creates a new `Administration` section with two pages: the first documents new features and the second discusses how to upgrade to 0.11 from versions <= 0.4

(Please excuse the unappealing commit history)

### CODE REVIEW

File: content/chronograf/v0.11/introduction/templating.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+title: Templating
+
+menu:
+  chronograf_011:
+    name: Templating
+    weight: 0
+    parent: Introduction
+---
+
+Easily modify the time range and [tag values](/influxdb/v0.10/concepts/glossary/#tag-value) displayed by a visualization and/or dashboard.
```
Comment (mark-rushakoff) [inline content/chronograf/v0.11/introduction/templating.md]: Should be 0.11 link?

File: content/chronograf/v0.11/introduction/getting_started.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+    ```
+
+    Then to load homebrew/binary/chronograf now:
+
+    ```
+launchctl load ~/Library/LaunchAgents/homebrew.mxcl.chronograf.plist
+    ```
+
+#### Standalone OS X binary
+Assuming you're working with Chronograf version 0.10, from the `chronograf-0.10/` directory:
```
Comment (mark-rushakoff) [inline content/chronograf/v0.11/introduction/getting_started.md]: Also should be 0.11.

**Comment by mark-rushakoff** [comment]
There's a couple spots where there are 0.10 mentions/links that i think should be 0.11.

Besides that small detail, very well written. :+1:

**Comment by rkuchan** [comment]
Thank you, @mark-rushakoff. I changed the 0.10 reference in the Getting Started guide.

I kept the 0.10 links that reference the InfluxDB documentation because we don't have docs for InfluxDB 0.11 yet. I'll be updating those once we get those docs up.

**Comment by kostasb** [comment]
Telegraf 0.11 automatically starts on Ubuntu as part of the installation package, with this pull request:
https://github.com/influxdata/telegraf/pull/747

Cameron actually merged this commit to close #747 instead of merging the original PR:
https://github.com/influxdata/telegraf/commit/e4e7d7fbfc9dd244146d1ded02543f0775ac6988

I've tested and confirmed that the service does start automatically on Ubuntu and it doesn't start automatically on RPM-based (eg Centos), which is the desired behavior.

Maybe we should add a note for it under the "Start the Telegraf Server" section.

**Pull request merged**